### ISSUE TITLE
[Documentation] Missing migration guide content: v19.x.x to v20.0.0

### ISSUE THREAD

**Issue by mpawlow** [i18next/i18next#1591]
## 🐛 Bug Report

- `Problem`: The migration guide is missing the corresponding v19.x.x to v20.0.0 content even though a major (semver) release has occurred
   - https://www.i18next.com/misc/migration-guide
- `Expected`: The migration guide to contain relevant upgrade information similar to what is already in the CHANGELOG
   - https://github.com/i18next/i18next/blob/master/CHANGELOG.md
```
Package  Current  Wanted  Latest  Location            Package Type  Homepage
i18next   19.9.2  19.9.2  20.1.0  dsx-service-broker  dependencies  https://www.i18next.com
```

## To Reproduce

- N/A

## Expected behavior

- N/A

## Your Environment

- N/A

**Comment by adrai** [comment]
There is no migration guide for v19 to v20.
There should not be a breaking change, but regarding of some misuse of i18next that pop up in last minor releases, we opted for a major version this time.

**Comment by mpawlow** [comment]
- Okay, makes sense.
- I guess the only issue is discoverability in the sense that I had to really search what the major version change was for:
   - e.g. Info not found in releases: https://github.com/i18next/i18next/releases
   - e.g. Info not found in migration guide: https://www.i18next.com/misc/migration-guide
- I actually only remembered to look into the CHANGELOG while authoring this bug report 🙂
